spec/services/form_validator_spec.rb

```diff
@@ -50,7 +50,7 @@
 
   context "with email as data_type" do
     let(:element) { create :form_element, :email, form: form }
-    let(:params){ {form_id: element.form_id, email: 'foo@example.com' } }
+    let(:params){ {form_id: element.form_id, email: "foo@example.com".force_encoding('ASCII-8BIT')} }
```

Comment:
why are you forcing ASCII always? shouldn't it just be in the `with ascii` example?